### PR TITLE
[RW-5791][risk=no] Avoid rotating status icon offscreen

### PR DESCRIPTION
**Pull request by calbach**
Issue: margin-left on the rotating icon went offscreen, causing a horizontal scrollbar to appear.

Fix: put the margin on a container div instead, let the SVG rotate without margin.

### CODE REVIEW

File: ui/src/app/components/help-sidebar.tsx [hunk 1]
```diff
@@ -111,14 +111,18 @@ const styles = reactStyles({
...
-    alignSelf: 'flex-end'
+    height: '.5rem'
   },
   runtimeStatusIconOutline: {
     border: `1px solid ${colors.white}`,
     borderRadius: '.25rem',
   },
+  runtimeStatusIconContainer: {
+    // This height properly aligns the status indicator vertically.
+    height: '.9rem',
```
Comment (calbach) [inline ui/src/app/components/help-sidebar.tsx:122]: There's probably a cleaner way of doing this. I wasn't really able to understand how the  svg was positioning itself relative to the div, and it seemed like I was going down a bit of a rabbit hole. Let me know if you have another idea for achieving this.

File: ui/src/app/components/help-sidebar.tsx [hunk 2]
```diff
@@ -111,14 +111,18 @@ const styles = reactStyles({
...
-    height: '.5rem',
-    margin: '0px .1rem .1rem auto',
-    alignSelf: 'flex-end'
+    height: '.5rem'
   },
   runtimeStatusIconOutline: {
     border: `1px solid ${colors.white}`,
     borderRadius: '.25rem',
   },
+  runtimeStatusIconContainer: {
```
Comment (als364) [inline ui/src/app/components/help-sidebar.tsx:120]: You can get rid of the height with the following changes here:

```suggestion
  runtimeStatusIconContainer: {
    alignSelf: 'flex-end',
    margin: '0 .1rem .1rem auto'
  }
```

and then changing the div you added to a FlexRow.

Comment (calbach) [reply ui/src/app/components/help-sidebar.tsx:120]: ah... much better - thanks!

**Comment by calbach** [comment]
@als364 PTAL, this still needs your approval

**Review by als364 — APPROVED** [review]
ope my bad! lgtm

**Pull request merged**